src/sync/installation.ts

```diff
@@ -239,6 +239,13 @@ const doProcessInstallation = async (data: BackfillMessagePayload, sentry: Hub,
 
 	const execute = async (): Promise<TaskPayload> => {
 		const gitHubInstallationClient = await createInstallationClient(gitHubInstallationId, jiraHost, logger, data.gitHubAppConfig?.gitHubAppId);
+		//
+		// We are not using "big" numbers here (e.g. 100) to speed up the process; API returns smaller pages faster and
+		// they will more likely not to break.
+		//
+		// Also this logic is broken for any rest queries (where cursor is the page number), because when the page size
```

Comment:
If this is broken, do we have a ticket in the backlog to address it?

Reply:
No, but I think I'll be touching this code soon as a part of hot-102548